WolvenKit.Modkit/RED4/RedMod.cs

```diff
@@ -0,0 +1,25 @@
+using WolvenKit.Common.Model.Arguments;
+
+namespace WolvenKit.Modkit.RED4
+{
+    public static class RedMod
```

Comment:
I have a feeling this could be somewhere else, but I don't know where to move these funcs for now.